index.csv

```diff
@@ -225,3 +225,5 @@ longest,MIT,1.0.1
 repeat-element,MIT,1.1.2
 trim,MIT,0.0.1
 css-value,MIT,0.0.1
+domutils,BSD-2-Clause,1.5.1
```

Comment:
this also applies to v1.1.1, v1.1.2, v1.1.3, v1.1.4, v1.1.5, v1.1.6, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v1.4.2, v1.4.3, and v1.5.0 - can you add those entries?